test/integration/users_can_view_cars_by_location_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.